llvm/test/Transforms/PGOProfile/ctx-instrumentation.ll

```diff
@@ -335,6 +339,41 @@ define void @unreachable() {
 ;
   unreachable
 }
+
+@foo_alias = weak_odr unnamed_addr alias void (i32, ptr), ptr @foo
+
+define void @call_alias(ptr %a) {
+; INSTRUMENT-LABEL: define void @call_alias(
+; INSTRUMENT-SAME: ptr [[A:%.*]]) {
```

Comment:
Why do you need to split this across 2 lines? Can you drop the SAME line and move the contents to the LABEL line above?

Reply:
This is autogenerated with `update_test_checks`